src/Forms/SearchableDropdownTrait.php

```diff
@@ -139,7 +139,6 @@ public function getPlaceholder(): string
                 return $emptyString;
             }
         }
-        $name = $this->getName();
```

Comment:
Looks like this isn't used below, which I assume is why you've removed it? Please add a note in future when making changes like this so it's easier to know what's going on.

It's obvious in your IDE because it presumably gets greyed out, but seeing why a change like this was made in a git diff requires more double checking.